packages/rest/src/parser.ts

```diff
@@ -95,6 +92,16 @@ function buildOperationArguments(
 ): OperationArgs {
   const args: OperationArgs = [];
```

Comment:
Guess we don't need `args` anymore since we have `paramArgs` defined below

Reply:
`args` is for keeping all arguments, including `parameters` + `requestBody`
`paramArgs` is for keeping `parameters` only.